jupyter-extension/jupyterlab_pachyderm/pps_client.py

```diff
@@ -95,7 +94,11 @@ async def create(self, path: str, body: dict):
             )
 
         companion_repo = f"{config.pipeline.name}__context"
-        if companion_repo not in (item.repo.name for item in client.list_repo()):
+        for item in client.list_repo():
+            same_project = config.pipeline.project.name == item.repo.project.name
+            if (companion_repo == item.repo.name) and same_project:
+                break
+        else:
```

Comment:
indentation seems off here

Reply:
This is actually proper syntax for the under-utilized for/else block: https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops